data/korath/korath hails.txt

```diff
@@ -0,0 +1,1482 @@
+		"sapir'r"      # sapphire (mispronounced)
+		"uranume"      # uranium (mispronounced)
+		"herdwuud"     # hardwood (mispronounced)
+		"tungst'n"     # tungsten (mispronounced)
+		"sann'd"       # sand (mispronounced)
+		"qurts"        # quartz (mispronounced)
+		"ste'l"        # steel (mispronounced)
+		"pla'tk"       # plastic (mispronounced)
+
+phrase "exile humanika: tactical and strategic items"
+	word
+		"mapse of humanika spass"  # Maps of human space (mispronounced)
+		"tar maps"                 # Star maps (mispronounced)
+		"humanika flit moment dita" # Human fleet movement data (mispronounced)
+		"r'd wermhol resurch"      # Red wormhole research (mispronounced)
+		"rad holl resurch"         # Red [worm]hole research (mispronounced)
+		"wermhol resurch"          # Wormhole research (mispronounced)
+		"sampil o humanika tetch"  # [Examples|Samples] of human technology (mispronounced)
+		"inernal govment docmants" # internal government documents (mispronounced)
+		"humanika beakoff gupes"   # Information about human break-off groups (ie.: Remnant or Alphas) (mispronounced)
+		"medital resorch humanika" # Medical research relating to humans. (Perhaps to keep captives alive, design bioweapons, or simple curiosity.) (mispronounced)
+		"nerf gus"                 # Nerve gas, probably to develop counter-measures. (mispronounced)
+		"all yer data"             # all [of] your data (mispronounced)
+		"'avigatun lugs"           # navigation logs (mispronounced)
+
+
+# These show the Korath Exiles do still try to have a culture. All of
+# these are horribly mispronounced human words.
+phrase "exile humanika: cultural items"
+	word
+		"sculping suplis"          # Sculpting supplies (mispronounced)
+		"painbush n paint"         # Paintbrushes and paint (mispronounced)
+		"musik'l instr'mats"       # Musical instruments (mispronounced)
+		"dr'm set"                 # Drum set (mispronounced)
+		"Mosart musaic"            # Mozart music (mispronounced)
+		"tess set's"               # chess sets (mispronounced)
+		"fingre paint'n k't"       # finger-painting kit (mispronounced)
+		"all yer operas"           # all of your operas (mispronounced)
+		"sonettes n haukus"        # sonnets and haikus (mispronounced)
+		"color'd pencals"          # colored pencils (mispronounced)
+		"buulk fabr'c"             # bulk fabric (mispronounced)
+		"strang instraments"       # string instruments (mispronounced)
+		"rok gittar"               # rock guitar (mispronounced)
+
+
+# These are items that make you wonder what strange things happen
+# among the Korath that make them ask for this.
+phrase "exile humanika: strange items"
+	word
+		"kit'ns"                   # Kittens  (mispronounced)
+		"Hoy ac'rn cookees"        # Hai acorn cookies (mispronounced)
+		"toilat pepper"            # Toilet paper (mispronounced)
+		"geko"                     # Gecko (mispronounced)
+		"Dund'gns en Sheragi"      # Dungeons and Sheragi (mispronounced)
```

Comment:
I think we can go one better while preserving the clarity of the reference. Typical spoof queue here is "location and threat" (or vice versa) with the same starting letters. "Bunkers and Badasses," "Ogres and Oubliettes," and "Monsters and Mazes" all come to mind, though I know I've seen more.

I like using Sheragi as direct sub for dragon a lot as well, so how about something like:

Stockades and Sheragi?
Spaceports and Sheragi?
Sheragi and Starbases?
Sheragi and Shipyards?

We could perhaps break the location pattern and go with "Saryds and Sheragi"/"Saryds and Sheragi"?

Reply:
That does sound good, yea :)